sl

```diff
@@ -1,12 +1,46 @@
 #!/usr/bin/env bash
 
 VERSION="1.0.0-rc1"
-LANGUAGE_PATH="./language/target/simplelanguage-$VERSION-SNAPSHOT.jar"
-LAUNCHER_PATH="./launcher/target/launcher-$VERSION-SNAPSHOT.jar"
+
 MAIN_CLASS="com.oracle.truffle.sl.launcher.SLMain"
+MY_PATH="$(cd "$(dirname "$0")" && pwd -P)"
 
 #######################################################################
-#            Check if the language and launcher jars exist            #
+# Locations of the language and launcher jars as well as the java command are
+# different if I'm running from the repository or as a component in GraalVM
+#######################################################################
+GRAALVM_VERSION=$(grep "GRAALVM_VERSION" "$MY_PATH/../release")
+if [[ "$GRAALVM_VERSION" != "" ]]; then
+    LANGUAGE_PATH="$MY_PATH/../jre/languages/sl/simplelanguage.jar"
```

Comment:
We shouldn't need to reference simplelanguage.jar here. It should be picked up automatically.